website/docs/r/compute_fleet.html.markdown

```diff
@@ -0,0 +1,740 @@
+
+* `reboot_setting` - (Optional) Specifies the reboot setting for all `AutomaticByPlatform` patch installation operations. Possible values are `Always`, `IfRequired`, `Never` and `Unknown`. Changing this forces a new resource to be created.
+
+* `secret` - (Optional) One or more `secret` blocks as defined below. Changing this forces a new resource to be created.
+
+* `vm_agent_platform_updates_enabled` - (Optional) Whether to enable the virtual machine agent platform updates for the linux virtual machine in the Compute Fleet. Defaults to `false`. Changing this forces a new resource to be created.
+
+---
+
+A `local_storage_in_gib` block supports the following:
+
+* `max` - (Optional) The maximum value of local storage in GiB.
+
+* `min` - (Optional) The minimum value of local storage in GiB.
+
+---
+
+A `memory_in_gib` block supports the following:
+
+* `max` - (Optional) The maximum value of memory in GiB.
+
+* `min` - (Optional) The minimum value of memory in GiB.
+
+---
+
+A `memory_in_gib_per_vcpu` block supports the following:
+
+* `max` - (Optional) The maximum value of memory per vCPU in GiB.
+
+* `min` - (Optional) The minimum value of memory per vCPU in GiB.
+
+---
+
+A `network_bandwidth_in_mbps` block supports the following:
+
+* `max` - (Optional) The maximum value of network bandwidth in Mbps.
+
+* `min` - (Optional) The minimum value of network bandwidth in Mbps.
+
+---
+
+
+A `network_interface_count` block supports the following:
+
+* `max` - (Optional) The maximum value of network interface count.
+
+* `min` - (Optional) The minimum value of network interface count.
+
+*
+
+---
+
+A `os_disk` block supports the following:
+
+* `caching` - (Optional) The Type of caching which should be used for the internal OS Disk. Possible values are `ReadOnly` and `ReadWrite`. Changing this forces a new resource to be created.
+
+* `delete_option` - (Optional) Specify what happens to the os disk when the virtual machine is deleted. Possible values are `Delete` and `Detach`. Changing this forces a new resource to be created.
+
+* `diff_disk_option` - (Optional) Specifies the Ephemeral Disk Settings for the OS Disk. The only possible value is `Local`. Changing this forces a new resource to be created.
+
+* `diff_disk_placement` - (Optional) Specifies where to store the Ephemeral Disk. Possible values are `CacheDisk`, `NvmeDisk` and `ResourceDisk`. Changing this forces a new resource to be created.
+
+* `disk_encryption_set_id` - (Optional) The ID of the Disk Encryption Set which should be used to encrypt the OS Disk. Changing this forces a new resource to be created.
+
+* `disk_size_in_gb` - (Optional) The size of the internal OS Disk in GB, if you wish to vary from the size used in the image the Compute Fleet is sourced from. Changing this forces a new resource to be created.
```

Comment:
is this gb or gib?

memory_in_gib_per_vcpu is used elsewhere in this resource so is this different and not gib?

Reply:
After confirmation, the unit of disk/os size should be GiB. Fixed.